spec/requests/api/merge_requests_spec.rb

```diff
@@ -277,7 +280,7 @@
 
       it "should return 201 when target_branch is specified and for the same project" do
         post api("/projects/#{fork_project.id}/merge_requests", user2),
-        title: 'Test merge_request', target_branch: 'master', source_branch: 'stable', author: user2, target_project_id: fork_project.id
+        title: "Test merge_request", target_branch: "master", source_branch: "stable", author: user2, target_project_id: fork_project.id
```

Comment:
Line is too long. [136/80]<br>Align the parameters of a method call if they span more than one line.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.